### DISCUSSION TITLE
No way to find something I just commented on

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#92716] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The challenge: comment on some random issue here on GitHub. Some issue not in any repository related to you, nor an issue you created.

OK, now the next day try to somehow find the issue. Yes, assume all you know is you commented on an issue, but you forgot what repository etc.

Only way to find it: check your browser history. OK, found it.

Now the next day try the same experiment, but via the GitHub app.

Too bad, now there is no way to find it, until someone replies to it and you get notified, or one day Google indexes it and you can search for it with "site:github.com myname".

**Comment by jidanni** [comment]
Yes, if this were Facebook the Activity Log would have it.
No, https://github.com/settings/security-log does not have it.
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-an-overview-of-your-activity-on-your-profile not there either.

**Comment by jidanni** [comment]
Holy smokes I think I found it.
It's there on the left side panel of the GitHub homepage,
![20240121T120959](https://github.com/community/community/assets/1230959/b621822c-23ae-4eed-9f9c-b6c6ffe25fcd)
Under "Recent activity"

**Reply by jidanni** [reply]
In the app "recent activity"  is called "recent".

Anyway it's just those several lines, and after an item falls off the back, tough luck finding it ever again. Same no matter cell phone or desktop.

**Comment by jidanni** [comment]
Wait, I'm making progress with
https://github.com/search?q=commenter%3Ajidanni&type=Discussions&s=date_created&o=desc
https://github.com/search?q=commenter%3Ajidanni&type=Issues&s=date_created&o=desc

**Comment by AMDphreak** [comment]
Same pain here ? +1.

Losing track of something you just commented on is common, and the UI does not help: **Mentions** is in the filter menu, but **Commenter** is not, even though `commenter:` search works. That makes building a shareable ?my recent participation? link (hiring, reviews, handoffs) unnecessarily awkward.

Related discussion with the UI gap and use cases: https://github.com/orgs/community/discussions/202870

Please consider exposing Commenter (and Involves) in the filter dropdowns.

**Comment by iawaisahmd** [comment]
Agreed. The commenter: search qualifier is a good workaround, but it's not very discoverable, and it doesn't replace having a dedicated "My comments" or "My participation" view. Exposing commenter: (and involves:) in the search/filter UI would make it much easier to revisit discussions and issues you've participated in without relying on browser history or notifications.
